src/modules/doctor/doctor.controller.ts

```diff
@@ -46,4 +51,18 @@ export default class DoctorController {
   ): Promise<Doctor> {
     return this.doctorService.updateDoctor(id, doctorDto);
   }
+
+  @UseGuards(JwtPatchGuard)
```

Comment:
why you protect one route not all controller?

Reply:
fixed. add guard to all controller